src/api/parser/test/url-parser.test.js

```diff
@@ -21,42 +21,42 @@ describe('url-parser tests', () => {
     expect(urlParser(url, port)).toBe(expected);
   });
 
-  // test('urlParser with double ports', () => {
-  //   const result = urlParser(testUrlWithPort, testport);
-  //   expect(result).toBe(expectedReturn1);
-  // });
-
-  // test('urlParser with 1 string port', () => {
-  //   const result = urlParser(testUrlWithoutPort, testport);
-  //   expect(result).toBe(expectedReturn1);
-  // });
-
-  // test('urlParser with invalid url', () => {
-  //   const result = urlParser('', testport);
-  //   expect(result).toBe(null);
-  // });
-
-  // test('urlParser with url with no port and no port', () => {
-  //   const result = urlParser(testUrlWithoutPort);
-  //   expect(result).toBe(expectedReturn2);
-  // });
-
-  // test('urlPaser with port being a null value', () => {
-  //   const result = urlParser(testUrlWithoutPort, null);
-  //   expect(result).toBe(expectedReturn2);
-  // });
-
-  // test('urlPaser with port being an integer', () => {
-  //   const result = urlParser(testUrlWithoutPort, Number(testport));
-  //   expect(result).toBe(expectedReturn1);
-  // });
-
-  // test('urlPaser with port being an invalid string', () => {
-  //   const result = urlParser(testUrlWithoutPort, 'invalid');
-  //   expect(result).toBe(expectedReturn2);
-  // });
-  // test('urlPaser with an out of range port', () => {
-  //   const result = urlParser(testUrlWithoutPort, 999999);
-  //   expect(result).toBe(expectedReturn2);
-  // });
+  test('urlParser with double ports', () => {
+    const result = urlParser(testUrlWithPort, testport);
+    expect(result).toBe(expectedReturn1);
+  });
+
+  test('urlParser with 1 string port', () => {
+    const result = urlParser(testUrlWithoutPort, testport);
+    expect(result).toBe(expectedReturn1);
+  });
+
+  test('urlParser with invalid url', () => {
+    const result = urlParser('', testport);
+    expect(result).toBe(null);
+  });
+
+  test('urlParser with url with no port and no port', () => {
+    const result = urlParser(testUrlWithoutPort);
+    expect(result).toBe(expectedReturn2);
+  });
+
+  test('urlPaser with port being a null value', () => {
+    const result = urlParser(testUrlWithoutPort, null);
+    expect(result).toBe(expectedReturn2);
+  });
+
+  test('urlPaser with port being an integer', () => {
+    const result = urlParser(testUrlWithoutPort, Number(testport));
+    expect(result).toBe(expectedReturn1);
+  });
+
+  test('urlPaser with port being an invalid string', () => {
+    const result = urlParser(testUrlWithoutPort, 'invalid');
+    expect(result).toBe(expectedReturn2);
+  });
+  test('urlPaser with an out of range port', () => {
+    const result = urlParser(testUrlWithoutPort, 999999);
+    expect(result).toBe(expectedReturn2);
+  });
```

Comment:
These ones I've changed it [test.concurrent.each`table`](https://jestjs.io/docs/api#2-testconcurrenteachtablename-fn-timeout) to be more intuitive

Reply:
Makes sense, lemme take them out